CHANGELOG.md

```diff
@@ -14,10 +14,15 @@ See README.md on how to build the hipCUB documentation using Doxygen.
 - CMake functionality to improve build parallelism of the test suite that splits compilation units by
 function or by parameters.
 - New overload for `BlockAdjacentDifference::SubtractLeftPartialTile` that takes a predecessor item.
+- Benchmarks for `BlockShuffle`, `BlockLoad`, and `BlockStore`.
 ### Changed
 - Improved build parallelism of the test suite by splitting up large compilation units for `DeviceRadixSort`, 
 `DeviceSegmentedRadixSort` and `DeviceSegmentedSort`.
-- CUB backend references CUB and Thrust version 1.17.1.
+- CUB backend references CUB and Thrust version 1.17.2.
```

Comment:
I think these changes need to be in the ROCm 5.5.0 section in the changelog here?

Reply:
Yes, this will have to be updated.  I will update these entries now.  The rest of the PR looks good, all CI passed, so I will merge in once I update the changelog.